### ISSUE TITLE
Nomeclature update (cultural)

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#7678]
I've noticed recently a number of entries at the [Nomenclature for Museum Cataloging](https://page.nomenclature.info/arbre-tree.app) (formerly Nomenclature 4.0) have changed from what we currently have in Arctos. Some of this includes new categories or sub-categories and then the item names nested below these.

Is there a way I can help fix these so they're up-to-date rather than requested them as I come across them (e.g., "[Fishing, Hunting & Trapping T&E](https://page.nomenclature.info/parcourir-browse.app?id=3802&lang=en&ws=INT&wo=N)" has recently replaced [Fishing & Trapping T&E](https://arctos.database.museum/name/Fishing%20&%20Trapping%20T&E) described in Arctos.

Please let me know the best way to move forward in making a holistic update rather than piecemealing these. 

Assigning **high priority** as I'm currently unable to assign the proper identification to an item that should be in the Hunting Equipment subclassification, but this doesn't exist in Arctos' version of the Nomenclature.

**Comment by dustymc** [comment]
This needs discussion.

Changing names is not allowed (and cannot be), I think this is best handled by a new name and an updated ID, but ???

Judging solely by the strings, it seems that these would be different circumscriptions (eg like a new species description in bioland). Are there relevant type publications (or equivalent) which might help understand what's going on?

**Comment by AJLinn** [comment]
According to [their website](https://page.nomenclature.info/apropos-about.app?lang=en): "The Nomenclature for Museum Cataloging website is the most up-to-date version of the Nomenclature standard."

Likewise: "Nomenclature continues to be developed and maintained by the Nomenclature Committee, with updates being performed directly within the system. **Nomenclature is a living standard that will evolve to meet the needs of its users.** [my emphasis added] See our [additional resources](https://page.nomenclature.info/mode_d_emploi-user_guide.app#H3.4.5) for more information on Nomenclature and other museum collections documentation standards."

"Changing names is not allowed (and cannot be)" is not an approach that will work for cultural collections. The naming/classification strategies are not the same as biological taxonomies - we don't identify our collections with the same degree of authority and those names are not set in stone by any means.

Nomenclature uses their committee to keep the entries up to date, which apparently means expanding categories and sub-categories and classes, in addition to removing and changing offensive and harmful language as it is identified. Arctos must be able to follow the same update pattern if we're going to use this system for cultural collection classification (and I 100% support continuing the use of the system). 

This is their [integration page ](https://page.nomenclature.info/integration.app?lang=en)- not sure if things have changed since we first created the Arctos-based model of this.

Cultural collections need to be able to have our classification systems be adaptable as the authorities are updated. Otherwise we might as well not be using anything.

**Comment by Jegelewicz** [comment]
The Nomenclature in Arctos is 4.0 anything new should be added as a new source?

**Comment by AJLinn** [comment]
>  anything new should be added as a new source

In 2018 Nomenclature 4.0 updated its name to Nomenclature for Museum Cataloging according to [this page](https://page.nomenclature.info/apropos-about.app?lang=en#Heading3-What). It includes:

- all of Nomenclature 4.0;
- harmonized data and images from the Parks Canada Descriptive and Visual Dictionary of Objects (This includes French equivalents to Nomenclature terms, illustrations, bibliographic data and more. Parks Canada and the American Association for State and Local History [AASLH] agreed to harmonize their two standards in order to promote standardized cataloging practices throughout both countries. Nomenclature replaces the online version of the Parks Canada Descriptive and Visual Dictionary of Objects);
- French equivalents for all of Nomenclature, including categories, classes, subclasses (and definitions of each) and terms; and
- Canadian spelling for some English terms, as alternatives to the U.S. spelling.

If we follow @Jegelewicz 's suggestion add a new source being Nomenclature for Museum Cataloging, can it be updated as terms and categories change? Is this an internal policy that we just need to update for cultural collections?

**Comment by dustymc** [comment]
>policy

No, it's a necessary characteristic of shared taxonomy. Without that constraint, someone will notice that their A was accidentally entered as Z and, rather than fixing identifications, update the taxonomy (thereby updating, without a trail, your 13,876 A) to "fix" it. Pretty much all of our rules exist because something like that happened....

**Comment by AJLinn** [comment]
I guess I'm not understanding this characteristic. If the authority that our taxonomy is based on changes, how do we not also change our (manually-entered) taxonomy?

**Comment by Jegelewicz** [comment]
> manually-entered

Nomencalture 4.0 was not manually entered - I did a lot of work taking a resource and converting it to a format suitable for Arctos. I suggest creating a new taxonomy source that the ES collections can maintain - perhaps start with my original Nomenclature 4.0 (via Arctos), but then you guys can change whatever you want. Anyone can switch to your new taxonomy source as a first choice and then anyone who is using Nomenclature 4.0 (via Arctos) won't suddenly have changes made to their classifications. If they want new classifications, they can add the new taxonomic source.

**Comment by Jegelewicz** [comment]
My latest update to Nomenclature 4.0 is in the Arctos Shared Drive - https://docs.google.com/spreadsheets/d/1ELVbD7hCdi2YJakwqnmnog7OKmOaeEQO/edit?usp=sharing&ouid=115457622099601875930&rtpof=true&sd=true

**Comment by AJLinn** [comment]
> Nomencalture 4.0 was not manually entered - I did a lot of work taking a resource and converting it to a format suitable for Arctos.

Sorry Teresa, that's what I meant. It's not something that's being piped in from somewhere else. You had to do a lot of work, I know, to make it fit our structure. 

I'd be happy to be the safekeeper of the new source, which would be the Nomenclature for Museum Cataloging (as maintained by UAM:EH), and keep it up to date based on the changes implemented at the authority reflected on their website.

Clearly I'll need to go through some training with you to understand the back end of all this.

**Comment by AJLinn** [comment]
In our UAM Arctos User Group meeting yesterday, @camwebb showed us the difference between these external sources that are "(via Arctos)" and introduced the concept of creating something unique to our own collection. We wondered if creating a UAM Culture" might be a solution to my issue of the Nomenclature being periodically updated and our desire to modify if/when changes occur. Might this be a solution for my query @dustymc and @Jegelewicz ?

If so, I tried to watch as carefully as I could while Cam flew through the various tasks, but I might need some help setting up the initial bulkloader. Nothing urgent now, so I'm downgrading the status, but would love to continue the conversation about a "UAM Culture" classification/taxonomy system based on the Nomenclature for Museum Cataloging.

**Comment by Jegelewicz** [comment]
Also, maybe you should just take over management of Nomenclature 4.0 (via Arctos) [[ link ]](https://arctos.database.museum/info/ctDocumentation.cfm?table=cttaxonomy_source#nomenclature_4_0__via_arctos_)?
